### ISSUE TITLE
obs.curtailment_limit_effective documentation is unclear

### ISSUE THREAD

**Issue by EBoguslawski** [rte-france/Grid2Op#321]
## Documentation issue description

If you run the following code which simply executes a curtailement action and print two quantities :

```
import grid2op
from grid2op.Parameters import Parameters
from grid2op.Action import PlayableAction

# Create environement
p=Parameters()
p.LIMIT_INFEASIBLE_CURTAILMENT_STORAGE_ACTION = True
env_name  = "educ_case14_storage"
env = grid2op.make(env_name, test=True, action_class=PlayableAction, param=p)

# Initialize the environment
env.seed(0)
env.set_id(0)
env.reset()

# Apply a curtailment action
action = env.action_space({'curtail': [(2, 0.1), (3, 0.15), (4, 0.2)]})
obs, _, _, _ = env.step(action)

# Print two quantites
print("obs.curtailment_limit :", obs.curtailment_limit)
print("obs.curtailment_limit_effective :", obs.curtailment_limit_effective)
```
You obtain the following output : 

```
obs.curtailment_limit : [1.   1.   0.1  0.15 0.2  1.  ]
obs.curtailment_limit_effective : [1.  1.  0.1 0.  0.  1. ]
```
As you can see effective curtailment limits seem to be more strict than the initial curtailment limits for gen 3 and 4. This is strange because the LIMIT_INFEASIBLE_CURTAILMENT_STORAGE_ACTION parameter should on the contrary soften the effective curtailment action.

This doesn't match the actual documention for `curtailment_limit_effective`  which is "Limit (in ratio of gen_pmax) imposed on each renewable generator effectively imposed by the environment."

## Suggested modifications

Precise in the documentation how `curtailment_limit_effective` is calculated.

**Comment by BDonnot** [comment]
Thanks 😊

Can you check the code and propose a good wording if it's possible?

Thanks

**Comment by BDonnot** [comment]
(this has been addressed and will be part of grid2op 1.7.1 release)

**Comment by BDonnot** [comment]
(I re open it as it is not yet merged into the main branch, i'll close it when i'll release grid2op version 1.7.1)
